packages/angular_devkit/build_angular/package.json

```diff
@@ -69,17 +69,17 @@
     "undici": "7.2.1"
   },
   "peerDependencies": {
-    "@angular/compiler-cli": "^19.0.0 || ^19.1.0-next.0",
-    "@angular/localize": "^19.0.0 || ^19.1.0-next.0",
-    "@angular/platform-server": "^19.0.0 || ^19.1.0-next.0",
-    "@angular/service-worker": "^19.0.0 || ^19.1.0-next.0",
+    "@angular/compiler-cli": "0.0.0-ANGULAR-PEER-DEP",
+    "@angular/localize": "0.0.0-ANGULAR-PEER-DEP",
+    "@angular/platform-server": "0.0.0-ANGULAR-PEER-DEP",
+    "@angular/service-worker": "0.0.0-ANGULAR-PEER-DEP",
```

Comment:
This looks great, but might be worth seeing how this plays out in the pnpm workspace. No idea yet.

Reply:
I think since this is just setting peer deps we should be good.